manifests/tekton/tasks/base/cosa-init.yaml

```diff
@@ -49,9 +49,6 @@ spec:
             --variant $(params.variant) \
             $(params.repo)
 
-        # Update symlinks for default manifests
-        cosa update-variant default $(params.variant)
```

Comment:
Thanks! Is this change compatible with the use of `$(readlink -f src/config/manifest.yaml)` down below?

Reply:
Thanks, updated.